schema/vulntology-json-schema-1.0-draft.json

```diff
@@ -324,6 +331,7 @@
             "required": [
                 "id",
                 "requiresAttackTheatre",
+                "hasExploitedWeakness",
```

Comment:
This reverses a prior PR (usnistgov/vulntology#73) where we reduced required properties.